docs/quickstart/using-sdk.rst

```diff
@@ -226,6 +228,13 @@ The code for ``bob`` will be very similar, with the only difference being that `
 Running this application files using ``netqasm simulate`` prints the outcomes of the two nodes.
 Since by default no noise is used, their outcomes will always be equal.
 
+
+.. note::
+
+   You may see a warning coming from NetSquid saying ``WARNING: a [sic] expression handler missing``.
```

Comment:
Maybe add a separate block or paragraph with 'known issues'. I know from experience that I check such paragraph for solved issues once in a while. With these notes throughout the text it is more difficult to maintain them when there are problems solved.